CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# [4.3.0](https://github.com/algolia/instantsearch.js/compare/v4.2.0...v4.3.0) (2020-02-25)
+
```

Comment:
What's confusing here is that since the Helper is a dependency, we flag it as a "chore", but if the state management was directly in InstantSearch, we would have flagged it as a "fix".

This is why I think we shouldn't care about it being a "chore", it's a "fix" for the users.

I recommend being explicit about the bug fix in the changelog.

Wdyt?

```suggestion

This version fixes a case where refinements for a facet with a name that matches a substring of another facet could be cleared by mistake. The fix was addressed in the [helper of the library](https://github.com/algolia/algoliasearch-helper-js/pull/760).
```

Reply:
This is still a fix in a dependency, not in InstantSearch. We made a lot of dependency updates that fixes some bugs at different levels but we do not document them in the Changelog. Also, the fix is already available in the users using the CJS or ESM builds, making it confusing saying **this** specific version of InstantSearch fixes this bug, which is false.